tipocket-ctl/tpctl/argo.py

```diff
@@ -151,3 +152,18 @@ def gen_case_template(self):
 
     def _get_case_template_name(self):
         return self.case.meta.name
+
+
+class ArgoCronCase(ArgoCase):
+    def __init__(self, feature, case: CaseInstance, image, tidb_cluster,
+                 notify=False, notify_users=None, cron_params=None):
+        super().__init__(feature, case, image, tidb_cluster,
+                         notify, notify_users)
+        self.cron_params = cron_params
+
+    def gen_workflow(self):
+        workflow = super().gen_workflow()
+        workflow['kind'] = 'CronWorkflow'
+        for k, v in self.cron_params.items():
+            workflow['spec'][k] = v
```

Comment:
I think the cron workflow generation is wrong.

The spec should look like the following
```yaml
spec:
  schedule: "* * * * *"
  concurrencyPolicy: "Replace"
  startingDeadlineSeconds: 0
  workflowSpec:
    entrypoint: whalesay
    templates:
```

Reply:
In addition, we'd better add suffix `-cron` to the workflow name to distinguish **random** run and **daily** run

Reply:
I've corrected this.